packages/terra-clinical-label-value-view/lib/LabelValueView.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
+
+.terraClinical-LabelValueView {
+  border: 0;
+  padding: 0.1785rem;
+}
+
+.terraClinical-LabelValueView-label {
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 20px;
+}
+
+.terraClinical-LabelValueView-value {
+  font-size: 1.4285714285714286rem;
+  line-height: 1.4285714285714286rem;
```

Comment:
+1 To Tyler's suggestion of having a prop for value but keeping children, I would conditionally render the value wrapper.

The Button does something similar: https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L73

When trying to render null nothing happens.